ironcage/tests/test_checks.py

```diff
@@ -0,0 +1,27 @@
+from django.core.checks import Error
+from django.test import override_settings, TestCase
+
+from ..checks import env_vars_check
+
+
+class EnvVarsCheckTest(TestCase):
+    """Tests for env_vars_check."""
+    def test_unset_in_prod(self):
```

Comment:
Won't this fail if any of these are set, eg by a `.env` file?  Let's cross that bridge if/when we come to it.